tests/acceptance/expected-failures.txt

```diff
@@ -912,3 +912,7 @@ apiWebdavUpload2/uploadFileUsingOldChunking.feature:98
 apiWebdavUpload2/uploadFileUsingOldChunking.feature:99
 apiWebdavUpload2/uploadFileUsingOldChunking.feature:100
 apiWebdavUpload2/uploadFileUsingOldChunking.feature:101
+#
+# https://github.com/owncloud/ocis-reva/issues/372 Listing shares via ocs API does not show path for parent folders
+apiShareManagementBasic/createShare.feature:269
```

Comment:
`@toImplementOnOCIS` tag removed here https://github.com/owncloud/core/pull/37809 maybe the commit id must be advanced

Reply:
but this is not the same file as core, this is the local feature for reva.

Reply:
even if we change anything in core this PR is still green, because we are already expecting the failures.